js/script.js

```diff
@@ -0,0 +1,65 @@
+var datArr=[];
+var count = 0;
+var interval = -1;
+var forEachChild = function(reddit){
+	reddit.data.children.forEach(function(children){
```

Comment:
Think about how to do this with filter and map.

It's better to use built-in language features to do things for comprehensibility's sake. If another programmer comes along and has to understand your code, they can look at a built-in feature and instantly figure out what it does. With something custom, that means they have to look at what you're doing to figure out wjhat's going on.